ui/src/pages/publications/[id]/index.tsx

```diff
@@ -3,10 +3,13 @@ import React from 'react';
 import * as Types from '@/types';
 
 export const getServerSideProps: Types.GetServerSideProps = async (context) => {
-    const requestedId = context.query.id;
+    const { id: requestedId, ...rest } = context.query;
+    const otherQueryParams = Object.keys(rest)
```

Comment:
No, not really, but I was ending up with just "?" being appended to the URL and this was my way of stopping that from happening.

Reply:
Could confirm some of them are populated, then forward the query? Otherwise make the call without.

I'm saying this like it's bad code :joy: I like to find alternatives.

Reply:
I think that's what this is doing, right? It filters the list to only ones with a value (via the !!) and only adds it to the URL if that filtered list has a length.